docs/layouts/absolute-layout.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebarDepth: 0
```

Comment:
It would be nice to improve documentation texts on this page. 
You can check https://nativescript-vue.org/en/docs/elements/layouts/absolute-layout/